src/integration-tests/src/secret_manager/openapi.rs

```diff
@@ -39,22 +39,20 @@ pub async fn run(tracing: bool) -> Result<()> {
 
     println!("\nTesting create_secret()");
     let create = client
-        .create_secret(
-            smo::model::CreateSecretRequest::default()
-                .set_project(&project_id)
-                .set_secret_id(&secret_id)
-                .set_request_body(
-                    smo::model::Secret::default()
-                        .set_replication(
-                            smo::model::Replication::default()
-                                .set_automatic(smo::model::Automatic::default()),
-                        )
-                        .set_labels(
-                            [("integration-test", "true")]
-                                .map(|(k, v)| (k.to_string(), v.to_string())),
-                        ),
+        .create_secret()
+        .set_project(&project_id)
+        .set_secret_id(&secret_id)
+        .set_request_body(
+            smo::model::Secret::default()
+                .set_replication(
+                    smo::model::Replication::default()
+                        .set_automatic(smo::model::Automatic::default()),
+                )
+                .set_labels(
+                    [("integration-test", "true")].map(|(k, v)| (k.to_string(), v.to_string())),
```

Comment:
Again nothing to do with this PR, but I wonder if this setter could be broadened so the code is valid without the mapping.

Reply:
Maybe if it consumed `IntoIterator` ?  We have #461 already, also created #477.